compilers/concrete-compiler/compiler/include/concretelang/Support/V0Parameters.h

```diff
@@ -67,20 +67,19 @@ struct V0Parameter {
 };
 
 namespace optimizer {
-constexpr double DEFAULT_GLOBAL_P_ERROR = 1.0 / 100000.0;
-constexpr double UNSPECIFIED_P_ERROR = NAN; // will use DEFAULT_GLOBAL_P_ERROR
-constexpr double UNSPECIFIED_GLOBAL_P_ERROR =
+const double DEFAULT_GLOBAL_P_ERROR = 1.0 / 100000.0;
```

Comment:
Strictly, because a `vector` can't be a `constexpr`, and I needed to add a default vector. But also because we don't need `constexpr` here from what I understand. Indeed, those would only be needed if we were performing computation on those values _during_ compilation. Something we don't do nor need I think.